main.go

```
@@ -220,6 +219,25 @@ func handleIndex(w http.ResponseWriter, r *http.Request) {
 }
 
+func verifyTurnstile(token, remoteIP string) (bool, error) {
+	resp, err := http.PostForm("https://challenges.cloudflare.com/turnstile/v0/siteverify", url.Values{
+		"secret":   {c.CaptchaSecret},
+		"response": {token},
+		"remoteip": {remoteIP},
+	})
+	if err != nil {
+		return false, err
+	}
+	defer resp.Body.Close()
```

golangci-lint (2.11.4)

[error] Error return value of `resp.Body.Close` is not checked

(errcheck)